### PR TITLE
Refine Experimental Message for AI Features

### LINKED ISSUE #14185: [Theia AI] Refine Experimental Message in IDE (142)
- [ ] In the chat, it still mentions the quick fix feature, but it does not mention the chat view, the command and the workspace agent
- [ ] It should link the online documentation (in the chat, in the settings and on the welcome page)
- [ ] The "cost" warning should be a bit stronger (chat, settings, welcome)

see also https://github.com/eclipse-theia/theia-blueprint/issues/398

### PR DESCRIPTION
**Pull request by JonasHelming**
fixed #14185
- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/ai-core/src/browser/ai-core-preferences.ts [hunk 1]
```diff
@@ -27,15 +27,16 @@ export const aiCorePreferenceSchema: PreferenceSchema = {
...
+            undergo significant changes. It is important to be aware that these experimental features may generate\
+            continuous requests to the language models (LLMs) you provide access to. This might incur costs that you\
+            need to monitor closely. By enabling this option, you acknowledge these risks.\
             \n\
-            By enabling this option, you acknowledge these risks and agree to provide feedback to help us improve.\
-            &nbsp;\n\
             **Please note! The settings below in this section will only take effect\n\
-            once the main feature setting is enabled.**',
+            once the main feature setting is enabled. After enabling the feature, you need to configure at least one\
+            LLM provider below. Also see [the documentation](https://theia-ide.org/docs/user_ai/)**',
```
Comment (planger) [inline packages/ai-core/src/browser/ai-core-preferences.ts:39]:
```suggestion
            LLM provider below. Also see [the documentation](https://theia-ide.org/docs/user_ai/)**.',
```

File: packages/ai-chat-ui/src/browser/chat-tree-view/chat-view-tree-widget.tsx [hunk 2]
```diff
@@ -147,11 +147,10 @@ export class ChatViewTreeWidget extends TreeWidget {
                     <div className="section-content">
                         <p>To enable the experimental AI features, please go to &nbsp;
                             {this.renderLinkButton('the settings menu', CommonCommands.OPEN_PREFERENCES.id)}
-                            &nbsp;and locate the <strong>Extensions &gt; ✨ AI Features [Experimental]</strong> section.</p>
+                            &nbsp;and locate the <strong>AI Features</strong> section.</p>
                         <ol>
                             <li>Toggle the switch for <strong>'Ai-features: Enable'</strong>.</li>
-                            <li>Provide an OpenAI API Key through the <strong>'OpenAI: API Key'</strong> setting or by
-                                setting the <strong>OPENAI_API_KEY</strong> environment variable.</li>
+                            <li>Provide at least one LLM provider (e.g. OpenAI), also see <a href="https://theia-ide.org/docs/user_ai/" target="_blank">the documentation</a> for more information</li>
```
Comment (planger) [inline packages/ai-chat-ui/src/browser/chat-tree-view/chat-view-tree-widget.tsx:153]:
```suggestion
                            <li>Provide at least one LLM provider (e.g. OpenAI), also see <a href="https://theia-ide.org/docs/user_ai/" target="_blank">the documentation</a> for more information.</li>
```

File: packages/ai-chat-ui/src/browser/chat-tree-view/chat-view-tree-widget.tsx [hunk 3]
```diff
@@ -163,11 +162,19 @@ export class ChatViewTreeWidget extends TreeWidget {
...
+                                    <li>Universal Chat Agent</li>
+                                    <li>Workspace Chat Agent</li>
+                                    <li>Command Chat Agent</li>
+                                    <li>Orchestrator Chat Agent</li>
+                                </ul>
+                            </li>
                             <li>{this.renderLinkButton('AI History View', 'aiHistory:open')}</li>
                             <li>{this.renderLinkButton('AI Configuration View', 'aiConfiguration:open')}</li>
                         </ul>
+                        <p>See <a href="https://theia-ide.org/docs/user_ai/" target="_blank">the documentation</a> for more information</p>
```
Comment (planger) [inline packages/ai-chat-ui/src/browser/chat-tree-view/chat-view-tree-widget.tsx:177]:
```suggestion
                        <p>See <a href="https://theia-ide.org/docs/user_ai/" target="_blank">the documentation</a> for more information.</p>
```

File: packages/getting-started/src/browser/getting-started-widget.tsx [hunk 4]
```diff
@@ -406,27 +406,28 @@ export class GettingStartedWidget extends ReactWidget {
         return <div className='gs-container gs-experimental-container'>
             <div className='flex-grid'>
                 <div className='col'>
-                    <h3 className='gs-section-header'> 🚀 Theia AI [Experimental] is available! ✨</h3>
+                    <h3 className='gs-section-header'> 🚀 AI Support in the Theia IDE is available! [Experimental] ✨</h3>
                     <br />
                     <div className='gs-action-container'>
-                        Theia IDE now contains the experimental "Theia AI" feature, which offers early access to cutting-edge AI capabilities within your IDE.
+                        Theia IDE now contains the experimental AI support, which offers early access to cutting-edge AI capabilities within your IDE.
```
Comment (planger) [inline packages/getting-started/src/browser/getting-started-widget.tsx:412]:
```suggestion
                        Theia IDE now contains experimental AI support, which offers early access to cutting-edge AI capabilities within your IDE.
```

File: packages/getting-started/src/browser/getting-started-widget.tsx [hunk 5]
```diff
@@ -406,27 +406,28 @@ export class GettingStartedWidget extends ReactWidget {
...
                         <br />
                         <br />
                         Please note that these features are disabled by default, ensuring that users can opt-in at their discretion without any concerns.
```
Comment (planger) [inline packages/getting-started/src/browser/getting-started-widget.tsx:415]:
```suggestion
                        Please note that these features are disabled by default, ensuring that users can opt-in at their discretion.
```

File: packages/getting-started/src/browser/getting-started-widget.tsx [hunk 6]
```diff
@@ -406,27 +406,28 @@ export class GettingStartedWidget extends ReactWidget {
...
                             onClick={() => this.doOpenExternalLink(this.theiaAIDocUrl)}
                             onKeyDown={(e: React.KeyboardEvent) => this.doOpenExternalLinkEnter(e, this.theiaAIDocUrl)}>
-                            {'Theia AI Documentation'}
+                            {'the documentation'}
                         </a>.
                         <br />
                         <br />
-                        We encourage feedback, contributions, and sponsorship to support the ongoing development of the Theia AI initiative use our&nbsp;
+                        🚧 Please note that this feature is currently in development and may undergo frequent changes.
+                        We encourage feedback, contributions, and sponsorship to support the ongoing development of the AI support use our&nbsp;
```
Comment (planger) [inline packages/getting-started/src/browser/getting-started-widget.tsx:430]:
```suggestion
                        We welcome your feedback, contributions, and sponsorship! To support the ongoing development of the AI capabilities please visit the&nbsp;
```

File: packages/getting-started/src/browser/getting-started-widget.tsx [hunk 7]
```diff
@@ -435,20 +436,16 @@ export class GettingStartedWidget extends ReactWidget {
...
                     <div className='gs-action-container'>
-                        Let's dive in!<br /><br />
                         <a
                             role={'button'}
                             style={{ fontSize: 'var(--theia-ui-font-size2)' }}
                             tabIndex={0}
                             onClick={() => this.doOpenAIChatView()}
                             onKeyDown={(e: React.KeyboardEvent) => this.doOpenAIChatViewEnter(e)}>
-                            {'Open the Theia AI Chat View now to catch a first glimpse and learn how to begin! ✨'}
+                            {'Open the AI Chat View now to learn how to begin! ✨'}
```
Comment (planger) [inline packages/getting-started/src/browser/getting-started-widget.tsx:448]:
```suggestion
                            {'Open the AI Chat View now to learn how to start! ✨'}
```

**Review by planger — COMMENTED** [review]
Thank you for the excellent improvements! I added a few more inline suggestions.

**Comment by JonasHelming** [comment]
@planger Wonderful, accepted them all!

**Review by planger — COMMENTED** [review]
Thanks, looks good to me! :+1:

**Review by planger — APPROVED** [review]

**Pull request merged**